ext/rotoscope/extconf.rb

```diff
@@ -2,4 +2,6 @@
 require "mkmf"
 
 $CFLAGS << ' -std=c99 -Wall -Werror -Wno-declaration-after-statement'
+$defs << "-D_XOPEN_SOURCE=600 -D_POSIX_C_SOURCE=200112L"
```

Comment:
Yeah the reason I opted for both was that originally I was checking for the presence of these and OS X doesn't define `_POSIX_SOURCE`, but Linux does. I thought I was covering my bases by requesting both, but I think I misunderstood how this works. 

I'll push this change 👍